src/styles/index.css

```diff
@@ -23,6 +23,7 @@
 @import './selection';
 @import './strike.css';
 @import './table.css';
+@import './image.css';
```

Comment:
I can't imagine any other image type that would need styles. Any reason to not generalize? 🙇